src/utils/dataMappers.ts

```diff
@@ -66,16 +66,38 @@ type LocationKeyMapperInput = {
   };
 };
 
-export type LocationKeyMapperOutput = {
+export type PlaceInfo = {
```

Comment:
At this point it's good to extract all those types into a separate `src/types` folder. Types usually should be stored in separate folder/files unless they are 100% going to be used only in one file.